epos_interface:
  ros__parameters:
    MAX_ACCELERATION: 10000
    MAX_DECELERATION: 10000
    PROFILE_VELOCITY: 7000

    debug: true